src/include/units/unit.h

```diff
@@ -172,4 +177,20 @@ struct deduced_unit : downcast_child<Child, detail::deduced_unit<Dim, U, URest..
 //   using prefix_family = PT;
 // };
 
+template<Unit U, basic_symbol_text Symbol>
+struct alias_unit : U {
+  static constexpr bool is_named = true;
+  static constexpr auto symbol = Symbol;
+  using prefix_family = U::base_prefix_family;
+  using base_prefix_family = U::base_prefix_family;
```

Comment:
I am not sure that it is always the case that when a `named_unit` has `prefix_family != no_prefix` than the aliased one will also be allowed to be prefixed with this unit. Maybe there are some aliased units that do not have a sense with prefixes even if base used them? I would prefer to provide `prefix_family` directly for an `alias_unit` as we do for `named_unit`.

Reply:
I see. Would it be a good idea to have the ```prefix_family``` default to the base unit's ```prefix_family``` unless specified otherwise (default template argument)?

That would leave all the existing aliased units as they are, and upon encountering some aliased unit that is an exception, one would provide the correct ```prefix_family```.

Reply:
Actually, I hoped that we will be able to get rid of base_prefix_family ;-) I do not want to overcomplicate the types.

Reply:
Oh, alright.

Reply:
Just realized that ```hectare``` is an ```alias_unit``` to ```square_hectometre``` that does not accept a prefix at all - so you're right, it's not always the case.